pages/common/install.md

```diff
@@ -23,3 +23,7 @@
 - Copy files and apply access/modification times of source to the destination:
 
 `install --preserve-timestamps {{path/to/source}} {{path/to/destination}}`
+
+- Copy files and create the directories in the path if they don't exist:
```

Comment:
```suggestion
- Copy files and create the directories in the destination directory if they don't exist:
```

Reply:
It's not clear... too many `directories`

Reply:
```suggestion
- Copy files and create the directories in the destination path if they don't exist:
```